packages/react/examples/custom-data-table-state-manager-vite/src/misc/enums.js

```diff
@@ -15,7 +15,7 @@ export const TABLE_SIZE = {
   /**
    * Regular size.
    */
-  REGULAR: 'normal',
+  REGULAR: 'lg',
```

Comment:
I switch the size name here to remove the warning. But the size is the same on the app.

![Screenshot 2023-11-29 at 10 41 32](https://github.com/carbon-design-system/carbon/assets/52183462/85b802fc-69a5-48bc-8df4-425b18eecadf)